client/src/components/common/applicant-info-2.vue

```diff
@@ -361,11 +364,15 @@ export default class ApplicantInfo2 extends NameRequestMixin {
     }
   }
 
-  nextAction () {
+  async nextAction () {
+    newReqModule.mutateIsLoadingSubmission(true)
     this.validate()
     if (this.isValid) {
-      this.next()
+      await this.submit()
     }
+    // hang on to the loading state for a bit
+    // to prevent users clicking button again while next component displays
+    setTimeout(() => newReqModule.mutateIsLoadingSubmission(false), 1000)
```

Comment:
I hope this makes sense. It feels a bit kludgy but I found that I was able to click the Submit button during the period between `this.submit()` and when the next component displays. I think it's all the reactive architecture in this app that makes things slow to respond.